### PR TITLE
Form clean up

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?
Tidied up the forms on first and second pages.
Removed a lot of superfluos elements for accessibility.
Created variables file.
Fixed select for income section on second page. On Ipad this just looked like text, so made it look like a dropdown.
Created new class for radio inputs. If there are more than 2 options, they will be split into buttons.

# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] tested in Chrome
- [ ] tested in Firefox
- [ ] tested in IE
- [ ] tested in Safari
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

**Comment by Br3nda** [comment]
some stray <br>
image

File: src/components/FirstTimeApplicant/WizardFormFirstPage.js [hunk 1]
```diff
@@ -120,35 +120,28 @@ class WizardFormFirstPage extends React.Component {
 
           <section>
             <div className="arrow-box primary">
-              <div>
                 <Address onSelection={this.handleAddressSelection} />
 
                 <Field name="valuation_id" type="hidden" component={renderField}/>
                 {this.state.rates_bill && this.state.rating_year && <Fragment>
-                  <div className="calc-layout">
+                  <p className="Select-results">
```
Comment (Br3nda) [inline src/components/FirstTimeApplicant/WizardFormFirstPage.js]: uppercase i unusual in css class names in my experience. Does it mean something?

File: src/components/widgets/Income.jsx [hunk 2]
```diff
@@ -85,9 +85,9 @@ class Income extends React.Component {
     return {
       'options': {
         'en': [
-          {value: 'below', label: 'less then $' + this.formatDollars(this.state.maximum_income_for_full_rebate)},
-          {value: 'between', label: 'somewhere in the middle'},
-          {value: 'above', label: 'more than $' + this.formatDollars(this.state.minimum_income_for_no_rebate)}
+          {value: 'below', label: 'Less than $' + this.formatDollars(this.state.maximum_income_for_full_rebate)},
```
Comment (Br3nda) [inline src/components/widgets/Income.jsx]: wow. good typo fix

**Comment by Br3nda** [comment]
It's very easy to enter "2" dependants, then use the scroll wheel and change it to negative.
When that happens, there's no eligibility shown

**Comment by dlouise64** [comment]
@Br3nda true good point about the scroll wheel, I know a workaround for that!

**Comment by Br3nda** [comment]
it's looking so good now. The css niggles are disappearing.
Good addition of variables!

**Pull request merged**